### ISSUE TITLE
unpkg and jsdelivr mismatch

### ISSUE THREAD

**Issue by xmaysonnave** [ethereum/web3.js#3217]
Dear Friends,
I'm loading dynamically web3 and face numerous issues on the browser side.
https://unpkg.com/web3@1.2.2/dist/web3.js
and
https://cdn.jsdelivr.net/npm/web3@1.2.2/src/index.js
are different.
I use browserify and babel and only the version from unpkg is loading and working correctly.

The latest
https://cdn.jsdelivr.net/npm/web3@1.2.4/src/index.js
and
https://unpkg.com/web3@1.2.4/dist/web3.js
are also different.
The 1.2.4 from both sources are not loading correctly in my environment.
As I always face a require is not defined error.

I'm not an expert though and I've to admit that I'm a bit lost.
I also tried to dynamically load babel-polyfill with no success with the 1.2.4

To summarize :
- Why there is a difference between unpkg and jsdelivr from the same version
- How may I solve the require is not defined
- What is the source to dynamically load web3

Warmly

**Comment by nivida** [comment]
We do not explicitly define bundles for them currently. This will be improved with PR #3160.
